File: ace-home/src/main/java/com/acehome/entities/Debt.java
package com.acehome.entities;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity()
@Table(name="debt")
@Getter
@Setter
@AllArgsConstructor
public class Debt {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column()
    private String name;

    @Column()
    private String description;

    @Column()
    private String creditorName;

    @Column()
    private float initAmount;

    @Column()
    private float currentAmount;

    @Column()
    private float paymentAmount;

    @Column()
    private float rate;

    @Column(columnDefinition = "boolean default false")
    private Boolean _delete;

    public Debt(){
        _delete = false;
    }

}

File: ace-home-cli/src/app/models/debt.ts
export class Debt{
    id:number;
    name:String;
    description:String;
    creditorName:String;
    initAmount:number;
    currentAmount:number;
    paymentAmount:number;
    rate:number;

    constructor(){
        this.id=0;
        this.name="";
        this.description="";
        this.creditorName="";
        this.initAmount=0;
        this.currentAmount=0;
        this.paymentAmount=0;
        this.rate=0;
    }
}